### PR TITLE
Refactor search panel code

### PR DESCRIPTION
**Pull request by tavriaforever**
@tadatuta

### CODE REVIEW

File: src/blocks/common.blocks/search-form/_type/search-form_type_default.styl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.search-form_type_default .search-form__input
```
Comment (tadatuta) [inline src/blocks/common.blocks/search-form/_type/search-form_type_default.styl]: can be replaced with [input_width_available](https://ru.bem.info/libs/bem-components/v2.1.0/desktop/input/#width)

File: src/blocks/common.blocks/search-form/search-form.browser.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                inited: function () {
+                    this._input = this.findBlockInside('input', 'input');
+                }
+            }
+        },
+
+        activate: function () {
+            var _this = this;
+
+            setTimeout(function () {
```
Comment (tadatuta) [inline src/blocks/common.blocks/search-form/search-form.browser.js:14]: what is the reason for timeout here?

File: src/blocks/common.blocks/search-panel/search-panel.deps.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+(
```
Comment (tadatuta) [inline src/blocks/common.blocks/search-panel/search-panel.deps.js]: looks like everywhere else `{` is on the same line as `(`

File: src/blocks/common.blocks/search-form/_type/search-form_type_default.styl (hunk 1 above)
Comment (tavriaforever) [inline src/blocks/common.blocks/search-form/_type/search-form_type_default.styl]: ok

File: src/blocks/common.blocks/search-panel/search-panel.deps.js (hunk 3 above)
Comment (tavriaforever) [inline src/blocks/common.blocks/search-panel/search-panel.deps.js]: ok

File: src/blocks/common.blocks/search-form/search-form.browser.js (hunk 2 above)
Comment (tavriaforever) [inline src/blocks/common.blocks/search-form/search-form.browser.js:14]: It is necessary to put the focus after opening a panel and fix a jolt panel

**Comment by tavriaforever** [comment]
update

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**